tests/test_utils.py

```diff
@@ -1711,3 +1711,74 @@ def test_patch_pred_store_sf() -> None:
     assert len(store) == 3
     for annotation in store.values():
         assert annotation.geometry.area == 4
+
+
+def test_patch_pred_store_zarr(tmp_path: pytest.TempPathFactory) -> None:
+    """Test patch_pred_store and persists predicted output to a zarr file."""
+    # Define a mock patch_output
+    patch_output = {
+        "predictions": [1, 0, 1],
+        "coordinates": [(0, 0, 1, 1), (1, 1, 2, 2), (2, 2, 3, 3)],
+        "probabilities": [[0.1, 0.9], [0.9, 0.1], [0.4, 0.6]],
+        "labels": [1, 0, 1],
+    }
+
+    save_path = tmp_path / "patch_output" / "output.zarr"
+
+    store_path = misc.patch_pred_store_zarr(patch_output, save_path=save_path)
+
+    print("Zarr path: ", store_path)
+    assert Path.exists(store_path), "Zarr output file does not exist"
+
+
+def test_patch_pred_store_zarr_ext(tmp_path: pytest.TempPathFactory) -> None:
+    """Test patch_pred_store and persists predicted output and ensures the output file
```

Comment:
First line is always a short line describing the function.
Next line is empty 
Description is after that.

"""Summary line.

Description.

"""

Reply:
Please see this page for further examples: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html